.github/workflows/pr-test.yml

```diff
@@ -132,7 +124,7 @@ jobs:
           # directory.
           # The purpose of this is for the PR Review Companion to later
           # be able to use this raw diff file for the benefit of analyzing.
-          git diff --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} > $BUILD_OUT_ROOT/DIFF
+          wget https://github.com/${{ github.repository }}/compare/${{ env.BASE_SHA }}...${{ env.HEAD_SHA }}.diff -O $BUILD_OUT_ROOT/DIFF
```

Comment:
We can see the diff url in https://docs.github.com/rest/commits/commits#compare-two-commits

This may avoid an additional api call (making it harder to reach the limitation)